### PR TITLE
Upgrade GoCD to 16.4.0

### PR DESCRIPTION
**Pull request by doctoryes**
@feanil @macdiesel Please review

### CODE REVIEW

File: playbooks/roles/go-agent/defaults/main.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@
 GO_AGENT_SERVICE_NAME: "go-agent"
 GO_AGENT_USER: "go"
 GO_AGENT_GROUP: "{{ GO_AGENT_USER }}"
-GO_AGENT_VERSION: "16.1.0-2855"
+#GO_AGENT_VERSION: "16.1.0-2855"
```
Comment (feanil) [inline playbooks/roles/go-agent/defaults/main.yml]: remove commented line.

File: playbooks/roles/go-server/defaults/main.yml [hunk 2]
```diff
@@ -13,7 +13,8 @@
 GO_SERVER_SERVICE_NAME: "go-server"
 GO_SERVER_USER: "go"
 GO_SERVER_GROUP: "{{ GO_SERVER_USER }}"
-GO_SERVER_VERSION: "16.1.0-2855"
+#GO_SERVER_VERSION: "16.1.0-2855"
```
Comment (feanil) [inline playbooks/roles/go-server/defaults/main.yml]: Remove commented line.

**Comment by feanil** [comment]
👍  once commented lines have been removed.

**Comment by macdiesel** [comment]
^^ 👍

**Pull request merged**